cameraserver/src/main/java/edu/wpi/first/wpilibj/vision/VisionRunner.java

```diff
@@ -20,7 +20,7 @@
  *
  * @see VisionPipeline
  * @see VisionThread
- * @see edu.wpi.first.wpilibj.vision
+ * @see <a href="package-summary.html">vision</a>
```

Comment:
Javadoc doesn’t support links to packages?

Reply:
From what I can tell, it doesn't support links to its own package.